WebApp/MarkupPanel.ascx

```diff
@@ -103,20 +109,20 @@ limitations under the License.
       <div class="frm_row">
         <div class="frmLabel">
           <span class="checkBoxWrap">
-            <%--  <gpv:Input type="checkbox" ID="chkMarkupLock" CssClass="frmCheckbox" runat="server" disabled="disabled" Style="visibility: hidden" />--%>
+           <gpv:Input type="checkbox" ID="chkMarkupLock" CssClass="frmCheckbox" runat="server" disabled="disabled" Style="visibility: hidden" />
             <gpv:Label ID="labMarkupLock" runat="server" for="chkMarkupLock" Style="visibility: hidden" title="Keep others from editing"><span class="">Title</span></gpv:Label>
           </span>
         </div>
         <div class="frmField">
-          <input type="text" id="tblMarkupTitle" class="frmInput btnControlLock" disabled="disabled" placeholder="Title" title="Title of the current markup group" /><br />
+          <input type="text" id="tboMarkupTitle" class="frmInput btnControlLock" disabled="disabled" placeholder="Title" title="Title of the current markup group" /><br />
         </div>
       </div>
       <div class="frm_row">
         <div class="frmLabel">
           <span class="">Details</span>
         </div>
         <div class="frmField">
-          <textarea id="tblMarkupDetails" class="frmTextarea" placeholder="Details" title="Detail of the current markup group"></textarea>
+          <textarea id="tboMarkupDetails" class="frmTextarea" placeholder="Details" title="Detail of the current markup group"></textarea>
```

Comment:
is this id used any where? if not remove this from the element.

Reply:
this id is used . for updating Markup detail value